cartridge-cli.lua

```diff
@@ -1066,31 +1070,35 @@ local function generate_version_file(dest_dir, app_name, app_version, app_releas
              "shipped to the resulting package. ")
     end
 
-    if tarantool_is_enterprise() then
+    if pack_state.tarantool_is_enterprise then
         -- copy TARANTOOL and TARANTOOL_SDK versions from SDK version file
         local tarantool_dir = get_tarantool_dir()
         local tnt_version = fio.pathjoin(tarantool_dir, 'VERSION')
         if not fio.path.exists(tnt_version) then
             warn("can't open VERSION file from Tarantool SDK. SDK information can't be " ..
                 "shipped to the resulting package. ")
         else
-            version_file:write(fio.open(tnt_version):read())
+            version_file:write(read_file(tnt_version))
         end
     else
         -- write TARANTOOL version
         version_file:write(string.format('TARANTOOL=%s\n', _TARANTOOL))
     end
 
-    version_file:write(string.format("%s=%s-%s\n", app_name, app_version, app_release))
+    version_file:write(string.format(
```

Comment:
May be we should check that our data were successfully written to file?

Don't ignore errors